juju/juju
# Lock podspec to focal

This PR allows pod spec charms to stay locked to focal for operators allowing the rest of juju to move forward to jammy.

Pod spec charms will eventually be removed, until then, focal should be recent enough.

## QA steps

- Deploy kubeflow-lite, check they use focal charm base image.
- Deploy kubeflow-lite to 2.9, model migrate to 3.0, model upgrade, check they use focal charm base image.
- Deploy kubeflow-lite to 2.9, upgrade to 3.0, check they use focal charm base image.

## Documentation changes

Update pod spec documentation to note that pod spec charms from 3.0 are locked to focal.

## Bug reference

N/A
